### ISSUE TITLE
Find badass doors

### ISSUE THREAD

**Issue by nerdynucleon** [cloyne/hackerspace#25]

**Comment by cotnerca** [comment]
Mural? Replace? Add lock?

**Comment by emilynathan** [comment]
Once we start installation of some of the equipment, I think a lock is a must.

**Comment by mitar** [comment]
I would not put a lock to the space itself, but we have those second doors and can put expensive equipment there, also we have that hole space, and storage room. I think we have enough of closed space for putting equipment in.

**Comment by emilynathan** [comment]
Are you talking about where the E-waste currently resides?

**Comment by mitar** [comment]
No. Inside the hackerspace, there are those metal wire-based doors. We can be closing those off. And then on the left side of those doors there is a hole where we could be storing even more stuff.

**Comment by emilynathan** [comment]
Oh gotchagotchagotcha!
